wgsl/index.bs

```diff
@@ -6589,7 +6613,7 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
 
     | [=syntax/ident=]
 
-    | [=syntax/callable=] [=syntax/argument_expression_list=]
+    | [=syntax/attr_unchecked_uniformity=] ? [=syntax/callable=] [=syntax/argument_expression_list=]
```

Comment:
Couldn't we make this generic as `attribute ? callable argument_expression_list`?
 Other attributes specify where they must apply in the table rather than inside the syntax. I think this way of opt-out is among the better candidates but IMO it should not case that specifically in the rule.

Reply:
Done. thanks